packages/config/config/devices/0x027a/zen20.json

```
@@ -162,6 +162,143 @@
 			"#": "27",
 			"$import": "~/templates/master_template.json#led_indicator_three_options",
 			"defaultValue": 1
+		},
+		{
+			"#": "28",
+			"label": "Enable/Disable All Power Reports",
+			"valueSize": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable all power monitoring and reporting",
+					"value": 0
+				},
+				{
+					"label": "Enable all power monitoring and reporting",
+					"value": 1
+				}
+			]
```

According to https://www.support.getzooz.com/kb/article/303-zen20-power-strip-change-log/, those were added in firmware 2.2.

Also we should simplify the wording and make use of the enable/disable template
```suggestion
			"$if": "firmwareVersion >= 2.2",
			"$import": "~/templates/master_template.json#base_enable_disable",
			"label": "Power/Energy Monitoring",
			"defaultValue": 1
```

Make it similar for the individual channels, just with a different label: "Power/Energy Monitoring: Relay 1", "...Relay 2" and so on.